kingpin/actors/rightscale/settings.py

```diff
@@ -44,7 +47,14 @@ def is_retriable_exception(exception):
         return False
 
     # Boto exceptions should have a code attribute
-    return str(exception) not in not_retry_codes
+    log.debug('Comparing "%s" to "%s".' % (str(exception), not_retry_codes))
+    skip_retry = any(code in str(exception) for code in not_retry_codes)
```

Comment:
Doesn't this mean it would also catch `client` or `error`? That seems a bit too wide of a net to me?